pkg/reconciler/cache/replication/replication_controller.go

```diff
@@ -114,12 +116,12 @@ func NewController(
 				local:  localKcpInformers.Tenancy().V1alpha1().WorkspaceTypes().Informer(),
 				global: globalKcpInformers.Tenancy().V1alpha1().WorkspaceTypes().Informer(),
 			},
-			tenancyv1alpha1.SchemeGroupVersion.WithResource("synctargets"): {
+			workloadv1alpha1.SchemeGroupVersion.WithResource("synctargets"): {
 				kind:   "SyncTarget",
 				local:  localKcpInformers.Workload().V1alpha1().SyncTargets().Informer(),
 				global: globalKcpInformers.Workload().V1alpha1().SyncTargets().Informer(),
 			},
-			tenancyv1alpha1.SchemeGroupVersion.WithResource("locations"): {
+			schedulingv1alpha1.SchemeGroupVersion.WithResource("locations"): {
```

Comment:
didn't we say in slack to not replicate locations?

Reply:
I can remove it. Was mainly focused on fixing the typo.